hw3/02-charts.js

```diff
@@ -28,18 +28,44 @@ const borderColors = [
 
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-const renderChart = () => {
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    const housesCounts = {};
+    const individualVals = [];
+    const houses = [];
+    const counts = [];
+    for (const character of data) {
+      houses.push(character.family);
+    }
+    houses.forEach(function (x) {
+      housesCounts[x] = (housesCounts[x] || 0) + 1;
```

Comment:
I would add comments to some stuff that aren't straight forward.  I tried doing it more on my codes too but it is easily forgettable.